bedrock/mozorg/templates/mozorg/about/governance/policies/security/plugin-permission-policy.html

```diff
@@ -106,8 +106,8 @@ <h2>{{ _('Version 1.0') }}</h2>
 
 <p>
     {% trans
-        whitelist_email='href=mailto:pluginwhitelist@mozilla.com' %}
-    Please contact Mozilla at <a {{ whitelist_email }}>pluginwhitelist@mozilla.com</a> for more information about this policy and answers to related questions.
+        allowedlist_email='href=mailto:pluginwhitelist@mozilla.com' %}
```

Comment:
Note: I'm not able to change this email address, but perhaps we can find where this goes to and issue a new / different address?